ext/bg/js/dictionary.js

```diff
@@ -262,16 +286,15 @@ function dictTermsMergeByGloss(result, definitions, appendTo=null, mergedIndices
     }
 
     for (const definition of definitionsByGloss.values()) {
-        definition.only = [];
-        if (!utilSetEqual(definition.expression, resultExpressionSet)) {
-            for (const expression of utilSetIntersection(definition.expression, resultExpressionSet)) {
-                definition.only.push(expression);
-            }
+        const only = [];
+        const expressionSet = definition.expression;
+        const readingSet = definition.reading;
+        definition.only = only;
```

Comment:
Doesn't matter in practice in JS, but could be more clear if `only` was added to `definition` after it's built